### PR TITLE
EFFTECH-951 use standardrb

### PR DESCRIPTION
**Pull request by apneadiving**

### CODE REVIEW

File: spec/bing_ads_ruby_sdk/preprocessors/order_spec.rb [hunk 1]
```diff
@@ -12,33 +11,37 @@ def action
       SpecHelpers.wrapper(:customer_management, "SignupCustomer")
     end
 
-    let(:unordered_params) {{
-      "Account" => {
-        "Name" => "test account",
-        "CurrencyCode" => "EUR",
+    let(:unordered_params) {
```
Comment (danielmbarlow) [inline spec/bing_ads_ruby_sdk/preprocessors/order_spec.rb:14]: Aren't multi-line `let` blocks supposed to be `do ... end`?

Comment (danielmbarlow) [reply spec/bing_ads_ruby_sdk/preprocessors/order_spec.rb:14]: I can add a quick PR for that if you don't want to. My Editor has a macro for it

**Review by danielmbarlow — COMMENTED** [review]
Looks good to me mostly. I'll just post a quick PR for the let blocks.

File: spec/bing_ads_ruby_sdk/preprocessors/order_spec.rb (hunk 1 above)
Comment (apneadiving) [reply spec/bing_ads_ruby_sdk/preprocessors/order_spec.rb:14]: man its green, why, oh why would you post style changes?

Comment (apneadiving) [reply spec/bing_ads_ruby_sdk/preprocessors/order_spec.rb:14]: the whole point of the linter is to avoid discussions about this

**Review by danielmbarlow — APPROVED** [review]

**Pull request merged**